cmd/service.go

```diff
@@ -102,7 +102,12 @@ func (m *ServerMux) handleServiceSignals() error {
 			}
 			runExitFn(nil)
 		case serviceStop:
-			log.Println("Gracefully stopping... (press Ctrl+C again to force)")
+			// Extra space will prevent showing "^CExiting" and rather show "^C Exiting" instead.
+			log.Println(" Exiting")
```

Comment:
That is OK, because otherwise we will have to write separate signal handlers just for this.

In case you have better alternative solution we can go for it.